update-docker-images

```diff
@@ -42,6 +46,16 @@ def docker_build (repository, task, version)
   end
 end
 
+task :login do
+  sh("docker login -u '#{env('USERNAME')}' -p '#{env('PASSWORD')}' https://hub.docker.com/r/#{env('REPOSITORY')}")
```

Comment:
This is a dangerous thing to do to provide passwords over the command line. Can we use a docker API instead?

https://github.com/swipely/docker-api

```
Docker.authenticate!(username: '...', password: '...')
image = Docker::Image.get(...)
image.push
```